docs/content.en/docs/release-notes/_index.md

```diff
@@ -29,6 +29,7 @@ Information about release notes of Coco Server is provided here.
 - refactor: create chat & send chat api #739
 - chore: icon support for more file types #740
 - chore: replace meval-rs with our fork to clear dep warning #745
+- chore: handle mcp interface parameters #746
```

Comment:
update the release notes, as it is not only refactoring for mcp now.

Reply:
ok, updated